VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/NotificationsSettingsScreen/NotificationsSettingsScreen.tsx

```diff
@@ -100,26 +66,39 @@ const NotificationsSettingsScreen: FC<NotificationsSettingsScreenProps> = ({ nav
       </Box>
     )
   }
+
+  const loadingCheck = loadingPreferences || registeringDevice || settingPreference
+
   return (
-    <FeatureLandingTemplate backLabel={t('settings.title')} backLabelOnPress={navigation.goBack} title={t('notifications.title')}>
-      <Box mb={contentMarginBottom}>
-        {systemNotificationsOn ? (
-          <>
-            <TextView variant={'MobileBodyBold'} accessibilityRole={'header'} mx={gutter}>
-              {t('notifications.settings.personalize.heading')}
-            </TextView>
-            <TextView variant={'MobileBody'} accessibilityRole={'header'} mx={gutter} mt={condensedMarginBetween}>
-              {t('notifications.settings.personalize.text.systemNotificationsOn')}
-            </TextView>
-            {preferenceList()}
-          </>
-        ) : (
-          alert()
-        )}
-        <TextView variant={'TableFooterLabel'} mx={gutter} mt={condensedMarginBetween}>
-          {t('notifications.settings.privacy')}
-        </TextView>
-      </Box>
+    <FeatureLandingTemplate backLabel={t('settings.title')} backLabelOnPress={navigation.goBack} title={t('notifications.title')} waygate="WG_NotificationsSettingsScreen">
+      {hasError ? (
+        <ErrorComponent screenID={ScreenIDTypesConstants.NOTIFICATIONS_SETTINGS_SCREEN} />
```

Comment:
I really hate all the conditional rendering in here. I think we need to figure out something better here in the future

Reply:
Confirmed with Dylan a code upkeep task will reduce these.